File: server/mbservice/src/main/java/org/rliz/mbs/MbServiceApplication.java
package org.rliz.mbs;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.context.annotation.ComponentScan;

/**
 * Main class to start the application.
 */
@EnableAutoConfiguration
@ComponentScan("org.rliz.mbs")
public class MbServiceApplication {

    public static void main(String[] args) throws Exception {
        SpringApplication.run(MbServiceApplication.class, args);
    }
}

File: server/mbservice/src/main/kotlin/org/rliz/mbs/MbServiceApplication.kt
package org.rliz.mbs

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.context.annotation.ComponentScan

/**
 * Main class to start the application.
 */
@EnableAutoConfiguration
@ComponentScan("org.rliz.mbs")
class MbServiceApplication

fun main(args: Array<String>) {
    SpringApplication.run(MbServiceApplication::class.java, *args)
}
